.eslintrc

```
@@ -0,0 +1,31 @@
+{
+  "extends": [
+    "airbnb",
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:@typescript-eslint/recommended-requiring-type-checking"
+  ],
+
+  "parserOptions": {
+      "project": ["./tsconfig.json"]
+  },
+
+  "rules": {
+    "@typescript-eslint/indent": "off",
```

```suggestion
    "@typescript-eslint/indent": "off",
    "indent": "off",
     "@typescript-eslint/indent": [
        "error",
        4
      ],
```

опять же – предлагаю 4 пробела

---

ок=)